docs/source/implementations/sb_bagging.rst

```diff
@@ -0,0 +1,149 @@
+.. _implementations-sb_bagging:
+
+======================================================
+Sequentially Bootstrapped Bagging Classifier/Regressor
+======================================================
+
+In sampling section we have shown that sampling should be done by Sequential Bootstrapping.
+SequentiallyBootstrappedBaggingClassifier and SequentiallyBootstrappedBaggingRegressor extend sklearn's BaggingClassifier/Regressor by using Sequential Bootstrapping instead of random sampling.
+
+In order to build indicator matrix, Triple Barrier Events and price bars used to label training data set should be passed as parameters.
```

Comment:
Can we please add an example of what that formatting looks like? I am thinking of users that build their own custom functions but still want to use these models. A good example is a technical trading system which defines its entry and exits based on certain conditions. This trade would have many properties shared with 3B (Start and end dates).